lms/djangoapps/discussion/django_comment_client/tests/test_utils.py

```diff
@@ -1230,6 +1234,20 @@ def test_tab_settings(self, mock_get_ccx):
         with self.settings(FEATURES={'CUSTOM_COURSES_EDX': True}):
             assert not self.discussion_tab_present(self.enrolled_user)
 
+    @override_settings(DISCUSSION_MICROFRONTEND_URL="http://test.url")
+    @ddt.data(
+        (True, 'http://test.url/discussions/{}/'),
+        (False, '/courses/{}/discussion/forum/'),
+    )
+    @ddt.unpack
+    def test_tab_with_mfe_flag(self, mfe_enabled, tab_link):
+        """
+        Tests that the correct link is used for the MFE tab
+        """
+        discussion_tab = CourseTabList.get_tab_by_type(self.course.tabs, 'discussion')
+        with override_waffle_flag(ENABLE_DISCUSSIONS_MFE, mfe_enabled):
+            assert discussion_tab.link_func(self.course, reverse) == tab_link.format(self.course.id)
+
```

Comment:
I believe the discussion tab should include the MFE link if the provider we have set is “edX” provider? What if non edX provider is set in the `DiscussionConfiguration` model?`

Reply:
The Discussion tab is only used for the "edX" provider. For LTI providers, the `DiscussionLtiCourseTab` is used.

Reply:
Got it, that's a separate tab.